clientv3/clientv3util/example_key_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2017 The etcd Authors
```

Comment:
Could you this file `example_key_test.go`  to dir `clientv3` e.g `clientv3/example_key_test.go`? The reason for this is that the [main_test.go](https://github.com/coreos/etcd/blob/master/clientv3/main_test.go#L31) in dir `clientv3` will create a 3 nodes cluster before running example test so that the test will pass. Otherwise, the test will run against nothing and probably get a time out error.